eth/ethconfig/config.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The go-ethereum Authors
+// Copyright 2021 The go-ethereum Authors
```

Comment:
is it ok to change the copyright just because we move a file?

Reply:
not sure. I can change it back.